sensor_msgs/LaserScan LaserSensor
nav_msgs/Odometry OdometrySensor
